packages/shared/src/lib/featureValues.ts

```diff
@@ -9,3 +9,10 @@ export enum AuthVersion {
   V3 = 'v3',
   V4 = 'v4',
 }
+
+export enum ShareVersion {
+  Control = 'control',
```

Comment:
I generally always ask design to refactor to V1 instead of Control for simplicity down the line.

I'll raise if with product to see if we can uniform naming as I don't like to mix versions with control or other naming.

Reply:
I named it v1 at first, but then saw other features with control version

Reply:
Hmm probably legacy features from back when.

If you can please rename this one to V1 as control can be either one of the versions going forward (it's more a product term at this stage)

Reply:
Done 👍